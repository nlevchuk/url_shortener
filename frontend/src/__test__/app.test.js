import App from '../app';

describe('<App />', () => {
  it('should use HeaderBox and MainBox', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
