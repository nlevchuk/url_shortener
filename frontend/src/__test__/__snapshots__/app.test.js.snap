// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should use HeaderBox and MainBox 1`] = `
<div
  className="app"
>
  <HeaderBox />
  <MainBox />
</div>
`;
